src/include/miopen/solver.hpp

```diff
@@ -2321,6 +2321,18 @@ struct PerformanceConfigConvOclBwdWrw2
     std::string ToString() const;
 };
 
+template <int N_BATCH_LOOPS>
+bool PerformanceConfigConvOclBwdWrw2<N_BATCH_LOOPS>::operator==(
+    const PerformanceConfigConvOclBwdWrw2<N_BATCH_LOOPS>& other) const
+{
+    // clang-format off
+    return n_waves == other.n_waves
+        && read_size == other.read_size
+        && n_out_channels_per_tile == other.n_out_channels_per_tile
+        && n_out_channels_tiles == other.n_out_channels_tiles
+        && n_out_rows_in_lcl == other.n_out_rows_in_lcl; // clang-format on
+}
```

Comment:
Why was this code moved here?

Reply:
There were some issues with load order which cause a bunch of warnings to be issued when trying to call the GetAllSolvers function from AnySolver. This may have been solved by the declaration of the specializations for PerformanceConfigConvOclBwdWrw2. So, I'll check if this can be reverted and still compile cleanly.

Reply:
@averinevg successfully reverted without affecting compilation warnings.